workers.py

```diff
@@ -308,9 +310,56 @@ def worker_topology(topology_info: dict) -> None:
 
         logger.debug('%s: Done, exiting', thread.name)
 
+    def worker_host(info: dict) -> None:
+        """Download and forward the content."""
+        thread = current_thread()
+        logger.debug('%s: Worker started', thread.name)
+
+        identity = json.loads(base64.b64decode(b64_identity))
+        account_id = identity.get('identity').get('account_number')
```

Comment:
Do we need the account Id for anything apart from logging? As a unique identifier, the `b64_identity` would serve the same purpose as well.

Reply:
The `account_id` is being used as the [batch_id](https://github.com/ManageIQ/aiops-ai-library/pull/17/files#diff-738ac49b5cddf3f7ce9f1a17ec48f249R56) which is in turn the [upload data id](https://github.com/ManageIQ/aiops-ai-library/pull/17/files#diff-738ac49b5cddf3f7ce9f1a17ec48f249R88) in `outlier-detection` service.